redgenes/gene_annotations.py

```diff
@@ -253,11 +280,33 @@ def annotation_pipeline(df, tmpdir, kofamscan_profile, kofamscan_kolist):
                     uscore,
                     tscore,
                     start_fuzzy,
-                    end_fuzzy)
-                VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?);"""
+                    end_fuzzy,
+                    run_id)
+                VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?);"""
```

Comment:
...you could envision a similar type of object to support interaction with the CDS data. but again, not for this PR, but something that we perhaps should consider later on